rero/rero-ils-ui
# layout and records

__admin records: backport briev views and editor__

* Adds the brief views for all the resources.
* Configures the menu.
* Adds routes for the record resources.
* Backports the library and circulation policies editors.
* Fixes lint for all the backport components.

__admin layout: implement layout and tools__

* Adds a footer.
* Adds a new menu component for navigation.
* Adds a front page.
* Adds a user service to retrieve the logged user informations.
* Updates the app-config service to retrieve the backend settings.

Co-Authored-by: Johnny Mariéthoz <Johnny.Mariethoz@rero.ch>
Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>

**Note:** this PR does not pass the tests, I suspect that it is related to `npm link` and needs also some additional job for the custom editors. Moreover, it contains multiple commits, thus please check the commits individually.

### To Test

- clone the @rero/ng-core project with this PR: https://github.com/rero/ng-core/pull/20 or wait for the version 0.0.8
- build and link ng-core `ng build @rero/ng-core` => got to the dist lib project and run `npm link`
- clone this project run `npm i; npm link @rero/ng-core`
- run `npm run start-proxy` and check the interface
